examples/leaf/wiring/specs/govector.go

```diff
@@ -0,0 +1,44 @@
+package specs
+
+import (
+	"strings"
+
+	"github.com/blueprint-uservices/blueprint/blueprint/pkg/wiring"
+	"github.com/blueprint-uservices/blueprint/plugins/goproc"
+	"github.com/blueprint-uservices/blueprint/plugins/govector"
+	"github.com/blueprint-uservices/blueprint/plugins/http"
+	"github.com/blueprint-uservices/blueprint/plugins/simple"
+	"github.com/blueprint-uservices/blueprint/plugins/wiringcmd"
+	"github.com/blueprint-uservices/blueprint/plugins/workflow"
+)
+
+// Wiring specification demonstrates how to instrument an application using GoVector to propagate vector clocks and to create logs with vector clocks.
+// The wiring spec uses govector logger as the custom logger for processes.
+// Each service is deployed in a separate process with services communicating using HTTP.
```

Comment:
Perhaps link to the govector plugin.

Might be worth updating some of the other specs to link to the plugins they demonstrate too.